src/ostorlab/runtimes/local/models/models.py

```diff
@@ -412,18 +424,51 @@ def create(
         Returns:
             AgentArgument object.
         """
+
         agent_argument = AgentArgument(
             agent_id=agent_id,
             name=name,
             type=type,
             description=description,
-            value=value,
+            value=value
+            if isinstance(value, bytes) is True
+            else AgentArgument.to_bytes(type, value),
         )
         with Database() as session:
             session.add(agent_argument)
             session.commit()
             return agent_argument
 
+    @staticmethod
+    def to_bytes(
+        type: str, value: Optional[Union[int, float, str, bool, List, Dict]]
+    ) -> bytes:
+        """Convert the value to bytes."""
+        if type == "string":
+            return value.encode(encoding="utf-8")
+        elif type in ("boolean", "bool"):
+            return (1 if value is True else 0).to_bytes(1, byteorder="big")
+        elif type in ("number", "int", "float"):
+            return struct.pack("d", value)
+        elif type in ("array", "object"):
```

Comment:
am not sure about `object`?

Reply:
what about it ? 
![image](https://github.com/Ostorlab/oxo/assets/144013278/74c13f8a-3e65-4c7a-bd6b-8c72148dcaa8)